src/plugins/ui_stateful/clipboard.ts

```diff
@@ -130,17 +142,37 @@ export class ClipboardPlugin extends UIPlugin {
       case "PASTE_FROM_OS_CLIPBOARD": {
         this._isCutOperation = false;
 
-        this.copiedData =
-          cmd.clipboardContent.data ||
-          this.convertTextToClipboardData(cmd.clipboardContent.text ?? "");
-
+        const htmlData = cmd.clipboardContent.data;
+        // TODO: support multiple image import
```

Comment:
maybe not necessary to have 3 times the same TODO

Reply:
it's more about "where should we act when we want to support several images"